packages/databricks-vscode/resources/python/00-databricks-init.py

```diff
@@ -131,10 +128,77 @@ def fs(self, line: str, local_ns):
         return cmd(*args[1:])
 
 
+def is_databricks_notebook(py_file: str):
+    if os.path.exists(py_file):
+        with open(py_file, "r") as f:
+            return "Databricks notebook source" in f.readline()
+
+def strip_hash_magic(lines: List[str]) -> List[str]:
+    if len(lines) == 0:
+        return lines
+    if lines[0].startswith("# MAGIC"):
+        return [line.partition("# MAGIC")[2] for line in lines]
+    return lines
+
+def convert_databricks_notebook_to_ipynb(py_file: str):
```

Comment:
this code is becoming pretty complex. Let's think about how we can add some unit tests.